### PR TITLE
AB#39714 Fixed bug where `Run query` button would spin forever

### PR DESCRIPTION
**Pull request by jerryxihe**

### CODE REVIEW

**Review by lhao03 — APPROVED** [review]
noICE (～￣▽￣)～

File: tabs/src/components/app/views/QueryRunner.jsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export function QueryRunner(props) {
         setIsLoading(true);
         const queryParameters = query.substring(GRAPH_URL.length + graphVersion.length, query.length);
         const graphResponse = await makeGraphCall(requestType, requestHeaders, queryParameters, graphVersion, requestBody, locale);
+        setIsLoading(false);
```
Comment (acchiang) [inline tabs/src/components/app/views/QueryRunner.jsx]: what happens if the promise is unresolved? eg. if there's an error
does the setIsLoading(false) still execute?

**Comment by lhao03** [comment]
```
const graphResponse = await makeGraphCall(requestType, requestHeaders, queryParameters, graphVersion, requestBody, locale)
            .catch(() => {
                // set default body 
                setReponseState("400" + " " + "Bad request");
                setIsLoading(false);
            });
setIsLoading(false);
```

File: tabs/src/components/app/views/QueryRunner.jsx (hunk 1 above)
Comment (acchiang) [reply tabs/src/components/app/views/QueryRunner.jsx]: resolved offline :)

Comment (jerryxihe) [reply tabs/src/components/app/views/QueryRunner.jsx]: 059acb93dda1c369ab8318f4ba0e43e30d21f4b0 Handled it in a similar way to the Graph Explorer web app

**Review by acchiang — APPROVED** [review]

**Review by lhao03 — APPROVED** [review]
noice (◍•ᴗ•◍)

**Review by acchiang — APPROVED** [review]

**Pull request merged**